src/component_selector.jl

```diff
@@ -146,7 +146,7 @@ Use `optional_and_fns` to return a function that computes the conjunction of `ge
 and the given function
 """
 available_and_fn(fn::Union{Function, Nothing}, sys) =
-    optional_and_fns(Base.Fix1(get_available, sys), fn)
```

Comment:
@GabrielKS You'll want to review these changes. Feel free to disagree.

Reply:
I need to understand this change better too.